<launch>

<node pkg="joy" type="joy_node" name="joy_node">
	<param name="dev" value="/dev/input/js0" type="str" />
</node>
<node pkg="navi_teleop" type="teleop_gamepad" name="teleop_gamepad">
	<remap from="cmd_vel" to="drive/cmd_vel"/>
	<param name="scale_angular" value="2"/> 
	<param name="scale_linear" value="3.3"/> <!-- m/s -->
</node>


</launch>
